nginx.conf.sigil

```diff
@@ -0,0 +1,85 @@
+server {
```

Comment:
I tried to get the deployment working without creating our own nginx override but was not possible, becasue we need to forwared some things to the server so that it can work properly and the default config was not built for that